activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -418,14 +418,26 @@ def change_column_null(table_name, column_name, null, default = nil) #:nodoc:
         end
 
         # Adds comment for given table column or drops it if +comment+ is a +nil+
-        def change_column_comment(table_name, column_name, comment) # :nodoc:
+        #
+        # Passing a hash containing +:from+ and +:to+ will make this change
+        # reversible in migration:
```

Comment:
Is it intentional to duplicate in the doc?

Reply:
According to https://github.com/rails/rails/pull/35851#issuecomment-480061623, this should be documented.
But I don't know which one of removing existing doc or duplicating with parent is best.

Reply:
Unlike https://github.com/rails/rails/pull/35851#issuecomment-480061623, this is intentionally deduplicated in the doc.
Unless you have strong opinion to duplicate this in the doc, please keep it as is.